tools/ctl/ide/commands/commands.go

```diff
@@ -501,9 +501,16 @@ func NewBenchmarkCmds(app *tview.Application, rb *rebuilder.Rebuilder, modalFn m
 					slices.SortFunc(rebuilds, func(a, b rundex.Rebuild) int {
 						return strings.Compare(a.ID(), b.ID())
 					})
+					var extranious int
 					rebuilds = slices.DeleteFunc(rebuilds, func(r rundex.Rebuild) bool {
-						return !tracked[r.ID()]
+						if tracked[r.ID()] {
+							return false
+						} else {
+							extranious += 1
+							return true
+						}
 					})
+					log.Printf("Throwing away %d extranious rebuilds", extranious)
```

Comment:
while theoretically useful for debugging performance issues with the TUI, i'm not sure this is worth keeping around now that you've implemented the dedup.